src/containerapp/azext_containerapp/_sdk_models.py

```diff
@@ -2096,23 +2106,29 @@ class Ingress(Model):
         'fqdn': {'key': 'fqdn', 'type': 'str'},
         'external': {'key': 'external', 'type': 'bool'},
         'target_port': {'key': 'targetPort', 'type': 'int'},
+        'exposed_port': {'key': 'exposedPort', 'type': 'int'},
         'transport': {'key': 'transport', 'type': 'str'},
         'traffic': {'key': 'traffic', 'type': '[TrafficWeight]'},
         'custom_domains': {'key': 'customDomains', 'type': '[CustomDomain]'},
         'allow_insecure': {'key': 'allowInsecure', 'type': 'bool'},
-        'ipSecurityRestrictions': {'key': 'ipSecurityRestrictions', 'type': '[IPSecurityRestrictions]'},
+        'ip_security_restrictions': {'key': 'ipSecurityRestrictions', 'type': '[IPSecurityRestrictions]'},
+        'client_certificate_mode': {'key': 'clientCertificateMode', 'type': 'str'},
+        'cors_policy': {'key': 'corsPolicy', 'type': 'CorsPolicy'},
     }
 
     def __init__(self, **kwargs):
         super(Ingress, self).__init__(**kwargs)
         self.fqdn = None
         self.external = kwargs.get('external', False)
+        self.exposedPort = kwargs.get('exposed_port', None)
         self.target_port = kwargs.get('target_port', None)
         self.transport = kwargs.get('transport', None)
         self.traffic = kwargs.get('traffic', None)
         self.custom_domains = kwargs.get('custom_domains', None)
-        self.allow_insecure = kwargs.get('allow_insecure', None)
-        self.ipSecurityRestrictions = kwargs.get('ipSecurityRestrictions', None)
+        self.allow_insecure = kwargs.get('allow_insecure', False)
```

Comment:
behavior change?

Reply:
In SDK the default value is false.
Should I follow SDK default value?